introduction.md

```diff
@@ -0,0 +1,7 @@
+# Hi, my name is Joe and I'm learning about Github. #
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.